Sources/SwiftCompilerPluginMessageHandling/Diagnostics.swift

```diff
@@ -126,3 +135,14 @@ extension PluginMessage.Diagnostic {
     }
   }
 }
+
+extension TriviaPiece {
+  var isSpaceOrTab: Bool {
+    switch self {
+    case .spaces, .tabs:
+      return true
+    default:
+      return false
+    }
+  }
+}
```

Comment:
This is copied. 
Is this fine or should we move it to `TriviaPiece` and make it public?

Reply:
I’m adding this in https://github.com/apple/swift-syntax/pull/1558, so once that’s merged, I don’t think you need to add it anymore 😉